yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -122,28 +122,58 @@ Internet-Draft                 YANG Semver                  October 2023
    submodule has non-backwards-compatible (NBC) changes compared to its
    previous revision, and a scheme that uses the revision history as a
    lineage for determining from where a specific revision of a YANG
-   module or submodule is derived.  Additionally, section 3.4 of
-   [I-D.ietf-netmod-yang-module-versioning] defines a revision-label
-   which can be used as an alias to provide additional context or as a
-   meaningful label to refer to a specific revision.
-
-   This document defines a revision-label scheme that uses extended
-   semantic versioning rules [SemVer] for YANG artifacts (i.e., YANG
-   modules, YANG submodules, and YANG packages
-   [I-D.ietf-netmod-yang-packages] ) as well as the revision label
-   definition for using this scheme.  The goal being to add a human
-   readable revision label that provides compatibility information for
-   the YANG artifact without needing to compare or parse its body.  The
-   label and rules defined herein represent the RECOMMENDED revision
-   label scheme for IETF YANG artifacts.
+   module or submodule is derived.
+
+   This document defines a YANG extension that tags a YANG artifact
+   (i.e., YANG modules, YANG submodules, and YANG packages
+   [I-D.ietf-netmod-yang-packages] ) with a version identifier that
+   adheres to extended semantic versioning rules [SemVer].  The goal
+   being to add a human readable revision label that provides
```

Comment:
There are a few instances of revision label or label left in the doc.

Reply:
Fixed.